templates/etc/profile.d/tlog.csh.epp

```diff
@@ -59,7 +59,8 @@ if ( ! ($?TLOG_RUNNING) ) then
 
           if ( "$TLOG_PASSTHROUGH_CMD" == "" ) then
             # Work around Tlog bug: https://github.com/Scribery/tlog/issues/269
-            if ( `hostname -f >& /dev/null` ) then
+            hostname -f >& /dev/null
+            if ( $? == 0 ) then
```

Comment:
This would make a _slightly_ nicer fix:
```suggestion
            if hostname -f >& /dev/null; then
```

I've started doing this with success checks since reading the rationale in [shellcheck SC2181](https://github.com/koalaman/shellcheck/wiki/SC2181)